README.md

```diff
@@ -1,4 +1,4 @@
-<a href="http://valor-software.com/ngx-bootstrap/#/">
+<a href="https://valor-software.com/ngx-bootstrap/#/">
```

Comment:
Is this really needed?

Reply:
Up to you and the group. There's a push for web privacy so I thought it would be a good update to throw in.